source/dlangbot/app.d

```diff
@@ -50,7 +50,8 @@ void startServer(HTTPServerSettings settings)
         .post("/codecov_hook", &codecovHook)
         .post("/buildkite_hook", &buildkiteHook)
         .post("/agent_shutdown_check", &agentShutdownCheck)
-        .get("/contributor_stats", &contributorStats)
```

Comment:
Note, this will break existing links and will require changes on dlang.org.

Reply:
Ok, I can leave the name unchanged.